ds-prep-python-conditionals/main.py

```diff
@@ -0,0 +1,55 @@
+day_of_week = "saturday"
+
+if day_of_week == "saturday" or "sunday":
+  print("Have a good weekend")
+else:
+  print("It's a weekday!")
```

Comment:
You'll find that this outputs the wrong message on weekdays:

---

![image](https://user-images.githubusercontent.com/16143771/149637017-71edb05b-e0e7-45ef-81a8-fbbb13923f16.png)